suite-native/transactions/src/components/TransactionsList/TransactionIconSpinner.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import {
+    Canvas,
+    Circle,
+    vec,
+    SweepGradient,
+    useTiming,
+    useComputedValue,
+} from '@shopify/react-native-skia';
+
+import { Box } from '@suite-native/atoms';
+import { useNativeStyles } from '@trezor/styles';
+import { CSSColor } from '@trezor/theme';
+
+type TransactionIconSpinnerProps = {
+    radius: number;
+    color: CSSColor;
+    strokeWidth?: number;
+};
+
+export const TransactionIconSpinner = ({
+    radius,
+    color,
+    strokeWidth = 8,
```

Comment:
Currently, you are not changing this value anywhere so I wouldn't pass it as optional prop. Just use it directly

Reply:
fixed 11399f17d